docs/Model_Equations.rst

```diff
@@ -0,0 +1,152 @@
+======================
+Set of Model Equations
+======================
+
+Economic Dispatch
+-----------------
+
+Linear programming is a mathematical modelling and optimization technique for a system of a linear objective function subject to linear constraints. The goal of a linear programming problem is to find the optimal value for the objective function, be it a maximum or a minimum. The MVS is based on oemof-solph, which in turn uses Pyomo to create a linear problem. The economic dispatch problem in the MVS has the objective of minimizing the production cost by allocating the total demand among the generating units at each time step. The equation is the following:
+
+.. math::
+        min Z = \sum_i a_i \cdot CAP_i + \sum_i \sum_t c_{var,i} \cdot E_i(t)
+.. math::        
+        CAP_i \geq 0
+.. math::        
+        E_i(t) \geq 0  \qquad  \forall t
+        
+.. math::
+        i \text{: asset}
+
+        a_i \text{: asset annuity [currency/kWp/year, currency/kW/year, currency/kWh/year]}
+
+        CAP_i \text{: asset capacity [kWp, kW, kWh]}
+
+        c_{var,i} \text{: variable operational or dispatch cost [currency/kWh, currency/L]}
+
+        E_i(t) \text{: asset dispatch [kWh]}
+
+The annual cost function of each asset includes the capital expenditure (investment cost) and residual value, as well as the operating expenses of each asset. It is expressed as follows:
+
+.. math:: 
+        a_i = \left( capex_i + \sum_{k=1}^{n} \frac{capex_i}{(1+d)^{k \cdot t_a}} - c_{res,i} \right) \cdot CRF(T) + opex_i
+.. math:: 
+        CRF(T) = \frac{d \cdot (1+d)^T}{(1+d)^t - 1}
+        
+.. math::
+        capex_i \text{: specific investment costs [currency/unit]}
+
+        n \text{: number of replacements of an asset within project lifetime T}
+
+        t_a \text{: asset lifetime [years]}
+
+        CRF \text{: capital recovery factor}
+
+        c_{res,i} \text{: residual value of asset i at the end of project lifetime T [currency/unit]}
+
+        opex_i \text{: annual operational and management costs [currency/unit/year]}
+
+        d \text{: discount factor}
+
+        T \text{: project lifetime [years]}
+
+The CRF is a ratio used to calculate the present value of the the annuity. The discount factor can be replaced by the weighted average cost of capital (WACC), calculated by the user. The lifetime of the asset t_a and the lifetime of the project T can be different from each other; hence, the number of replacements n is estimated using the equation below:
```

Comment:
I feel like the text about CRF and wacc should be placed before the respective equations. Alternatively, you can add an empty line before starting the explanation around n